packer/install-dependencies.sh

```diff
@@ -431,8 +431,4 @@ cat ${RELEASE_NOTES_FILEPATH}
 echo "END_OF_NOTES"
 set -x
 
-# Move logs from VHD creation out of /var/log
-sudo mv /var/log /var/log.vhd
-sudo mkdir /var/log
```

Comment:
so actually applyCIS changes the permissions on files in the /var/log dir so we might not actually want to run this after applyCIS (sorry I know I'm the one who told you we should in the first place...) I'm not sure how the `cloud-init clean --logs` affects applyCIS / if it reverts setting the permissions on cloud init logs. If that's the case we might want to consider moving all logs permission changes to CSE or cloud init instead of the VHD script. What do you think?

Reply:
Let's move the /var/log stuff back and then we'll see what cloud-init cleanup does